app/src/controllers/ats.ts

```diff
@@ -12,6 +15,17 @@ const controller = {
     try {
       const response = await atsService.searchATSUsers(req.query);
 
+      res.status(response.status).json(response.data);
+    } catch (e: unknown) {
+      next(e);
+    }
+  },
+
+  createATSClient: async (req: Request<never, never, ATSClientResource, never>, res: Response, next: NextFunction) => {
+    try {
+      const atsClient = req.body;
+      atsClient.createdBy = `${IdentityProvider.IDIR.toUpperCase()}\\${getCurrentUsername(req.currentContext)}`;
```

Comment:
While it is only IDIR users that can access this API, we should aim to genericize as much as possible. You can pull the identity provider from the current context.